components/function-runtimes/python38/kubeless/kubeless.py

```diff
@@ -180,12 +180,22 @@ def preload():
         'function_failures_total', 'Number of exceptions in user function', ['method']
     )
 
-    loggedapp = requestlogger.WSGILogger(
-        app,
-        [logging.StreamHandler(stream=sys.stdout)],
-        requestlogger.ApacheFormatter(),
-    )
+    # added by Kyma team
+    if os.getenv('KYMA_INTERNAL_LOGGER_ENABLED'):
+        # default that has been used so far
+        loggedapp = requestlogger.WSGILogger(
+            app,
+            [logging.StreamHandler(stream=sys.stdout)],
+            requestlogger.ApacheFormatter(),
+        )
+    else:
+        loggedapp = app        
+    # end of modified section
+
 
+    quiet = 'KYMA_BOTTLE_QUIET_OPTION_DISABLED' not in os.environ
+    print(quiet)
+    print("ASDASD")
```

Comment:
Logs from python functions looks like:
```
kubectl logs -n pprecel test3-cmtgb-78f8cc878-g4rsw function -f
True
ASDASD
```
Is `print("ASDASD")` necessary? And can we add another annotation to this line `print(quiet)`? Maybe, `print("quiet mode:", quiet)` looks better, or just remove this line too. If you decide to improve arguments for the `print` method, maybe we can implement something similar to the node functions too?

Reply:
lol outdated 🥇